pkg/osquery/table/gdrive_sync.go

```diff
@@ -0,0 +1,61 @@
+func GDrivePlugin(client *osquery.ExtensionManagerClient) *table.Plugin {
+	paths := queryDbPath...
+	database, _ := sql.Open("sqlite3", path)
```

Comment:
should not ignore error. However not sure what to do about it, lets look at other example.